packages/sdk-snippets/README.md

```diff
@@ -87,8 +87,7 @@ app.post('/webhook', express.json({ type: 'application/json' }), async (req, res
   });
 });
 
-const port = 8000;
-const server = app.listen(port, '0.0.0.0', function () {
-  console.log('Example app listening at http://%s:%s', server.address().address, port);
+const server = app.listen(8000, '0.0.0.0', function () {
```

Comment:
You could also move to a lambda format if you wanted to: 
```suggestion
const server = app.listen(8000, '0.0.0.0', () => {
  console.log('Example app listening at http://%s:%s', server.address().address, server.address().port);
});
```

Reply:
Yesss into it! Now that you've pointed it out, I notice we use arrow syntax earlier in these snippets too. Thanks!